server/middlewares/PartyValidator.js

```diff
@@ -23,16 +40,16 @@ class PartyValidator {
         errors: helpers.extractErrors(errors),
       });
     }
-    // const partyExists = PartyModel.find(party => (
-    //   party.name.toLowerCase() === req.body.name.toLowerCase()
-    // ));
-    const partyExists = helpers.recordExists(PartyModel, req);
-    if (partyExists) {
-      return res.status(409).json({
-        status: 409,
-        error: 'The party already exists',
-      });
-    }
+    (async () => {
+      const party = await partyExists(req.body.name);
```

Comment:
The same as above [here](https://github.com/oriechinedu/Politico/pull/61/files#r253825787)